relay-server/src/processing/spans/process.rs

```diff
@@ -84,3 +85,472 @@ fn normalize_span(
 
     Ok(())
 }
+
+/// Applies PII scrubbing to individual spans.
+pub fn scrub(spans: &mut Managed<ExpandedSpans>, ctx: Context<'_>) {
+    spans.modify(|spans, records| {
+        spans.spans.retain_mut(|span| {
+            let r = scrub_span(span, ctx).inspect_err(|err| {
+                relay_log::debug!("failed to scrub pii from span: {err}");
```

Comment:
Should we log an error here? Seems like a "should not happen" situation. Or do we rely on error logs at the time the PII config is compiled?